src/pages/about.astro

```diff
@@ -103,12 +103,41 @@ const members = filterInactiveMembers(await getMembers());
       <section>
         <h2>What payments are eligible?</h2>
 
-        <p
-          >Companies directly pay maintainers of Open Source projects they depend on, with no strings attached. Paying
-          developers to do specific work does not count towards the Pledge. Feel free to participate in <a
-          href="https://github.com/opensourcepledge/osspledge.com/issues/112">our ongoing discussion</a> on what counts
-          and what doesn't.</p
-        >
+        <p>The primary intent with the Pledge is <b>no-strings-attached payments
+        to community projects</b>. Start there.</p>
+
+        <ol>
+
+          <li>Open Source projects that your company controls are out of scope.
+            If you own the trademark, it doesn't count.</li>
+
+          <li>Open Source projects that exclusively benefit your company or
+            your company's own ecosystem are out of scope.</li>
+
+          <li>It's okay if you get non-exclusive logo placement or other
+            acknowledgement for your payments, or an hour or two of consulting
+            time with the project team, but anything beyond such minimal
+            benefits puts the payment out of scope.</li>
```

Comment:
`an hour or two` should be in relation to the donation – the consulting time should be very minor in compared to the donation – a donation of `$100` to a project and a donation of `$10k` merits different minimal benefits

Reply:
Better?

```suggestion
          <li>It's okay if you get non-exclusive logo placement or other
            acknowledgement for your payments, or an hour or two of consulting
            time along with a large sponsorship, but anything beyond such minimal
            benefits puts the payment out of scope.</li>
```

Reply:
@chadwhitacre I would leave it out completely I think – if it isn't a minimal benefit / compensation then it doesn't matter if its one or ten hour – and depending on the donation one hour can be huge or ten hours can be small

If a company donates $10k monthly to a project, then eg. getting 10h per month of compensation is negligible

Also remember that without any compensatory benefits it can be hard to deduct the donation as a valid expense in some jurisdictions.

Reply:
It's a common case, I think it's fine to mention as an example.

Reply:
@chadwhitacre Like worded now it sounds like a $50 donation can be compensated with the same consulting hours as a $10k though?

Any $50 donation that gives consulting hours should be considered a payment, as its a normal hourly rate.

So the example confuses more than it clarifies?

Reply:
Are we talking about the same text?

> or an hour or two of consulting time along with a large sponsorship

How does this make it sound like a $50 sponsorship can be compensated with the same consulting hours as a $10k sponsorship?

Reply:
`large` is very hard to quantify – is it `$1k`, `$10k`, `$100k`? my point is that it should be relative to the donation, not fixed?

Reply:
Dude you're overthinking it, this illustrates the point that receiving minimal ancillary benefits doesn't discount payments that have the right intent.